ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -76,33 +77,88 @@ export interface AccountCreationState {
 }
 
 const styles = {
-  sectionLabel: {
-    height: '22px',
+  asideContainer: {
+    backgroundColor: colorWithWhiteness(colors.primary, 0.85),
+    borderRadius: 8,
+    height: '17rem',
```

Comment:
This is a container for text, so it all is relative to text sizing here. This page is going to be a lot more on absolute height and width I think, because of the nature of it being a pretty heavy weight form. (This one is probably the most changeable to percentages, but in general I wanted to stick with the theme of the page.)